src/ontology/foodon-edit.owl

```diff
@@ -57823,12 +57838,17 @@ SubClassOf(obo:FOODON_03460172 obo:FOODON_00002454)
 # Class: obo:FOODON_03460173 (food (salted))
 
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03460173 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03460173 "requires curation"@en)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03460173 "A food product which is output of a salting process where salt is dissolved or absorbed into the food as an ingredient or flavourant."@en)
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03460173 "http://langual.org")
+AnnotationAssertion(dc:contributor obo:FOODON_03460173 "https://orcid.org/0000-0001-5193-0062 | https://orcid.org/0000-0001-8008-8249"@en)
 AnnotationAssertion(oboInOwl:hasDbXref obo:FOODON_03460173 "http://www.langual.org/langual_thesaurus.asp?termid=H0173")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:FOODON_03460173 "salted"@en)
+AnnotationAssertion(oboInOwl:hasSynonym obo:FOODON_03460173 "food (compounded)"@en)
```

Comment:
This phrase wouldn't be found in text mining - I think the phrase you would want is "compounded food" ?

Reply:
Thanks for the tip! Have amended in later commit.